src/initialization/FGInitialCondition.h

```diff
@@ -443,6 +443,10 @@ class JSBSIM_API FGInitialCondition : public FGJSBBase
       @param wD Initial wind velocity in local down direction, feet/second   */
   void SetWindNEDFpsIC(double wN, double wE, double wD);
 
+  /** Sets the initial total wind speed.
+      @param mag Initial wind velocity magnitude in feet/second */
+  void SetWindMagFpsIC(double mag);
```

Comment:
As per my comment above.
```suggestion
  void SetWindMagFpsIC(double mag) { SetWindMagKtsIc(mag*fpstokts); }
```